IPython/core/interactiveshell.py

```diff
@@ -3585,7 +3585,9 @@ def enable_matplotlib(self, gui=None):
             make sense in all contexts, for example a terminal ipython can't
             display figures inline.
         """
-        from matplotlib_inline.backend_inline import configure_inline_support
+        # NOTE: this import triggers `enable_gui('qt')` and occurs before the users' config (e.g.
+        # `--matplotlib=qt5`) is procssed.
+        # from matplotlib_inline.backend_inline import configure_inline_support
```

Comment:
Removing this import (and the function call that requires it) prevents the unnecessary installation of a `qt` event loop hook, and as far as I can tell, has no negative effects - I still get inline figures in a Jupyter notebook, for example. I'll need someone with more expertise to chime in on this.